### PR TITLE
Matrix tests and benchmarks

### PR DESCRIPTION
**Pull request by idanmiara**

### CODE REVIEW

File: tests/matrix_tests/matrix_test.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+
+def test_parasym():
+    from parasymbolic_matrix import ParasymbolicMatrix
+    from parasymbolic_matrix.mock_parasymbolic_matrix import MockParasymbolicMatrix
+    from scipy_matrix import ScipyMatrix
+
+    MatrixA = ParasymbolicMatrix
+    MatrixB = MockParasymbolicMatrix
+    MatrixC = ScipyMatrix
```
Comment (bentheiii) [inline tests/matrix_tests/matrix_test.py:78]: use `import as`, and place at top of file

File: tests/matrix_tests/matrix_test.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    subtest_parasym(MatrixA, MatrixB)
+    subtest_parasym(MatrixA, MatrixC)
```
Comment (bentheiii) [inline tests/matrix_tests/matrix_test.py:80]: split into two tests

Comment (idanmiara) [reply tests/matrix_tests/matrix_test.py]: done

File: tests/matrix_tests/matrix_test.py (hunk 1 above)
Comment (idanmiara) [reply tests/matrix_tests/matrix_test.py]: done

**Comment by idanmiara** [comment]
I've refactor these tests.

**Comment by bgmoshe** [comment]
@idanmiara  What about this PR? Should it be closed ?
@bentheiii

**Pull request closed without merging**